src/lib/components/wizard-navigation-bar.component.vertical.less

```diff
@@ -31,368 +31,191 @@
     }
-  }
-
-  &.large-filled-symbols {
-    ul.steps-indicator {
-      padding: (@distance-between-steps / 2) (@distance-between-steps / 2) (@distance-between-steps / 2) (@distance-between-steps / 2 + @dot-baseline-distance + @big-dot-height);
-
-      li {
-        &:not(:last-child):before {
-          .line(@big-dot-width, @big-dot-height, @wz-color-default);
-        }
-
-        .step-indicator {
-          .state-circle(@big-dot-width, @big-dot-height, 0);
-          .state-circle-with-background-and-content(@wz-color-default);
-        }
-
-        a:hover .step-indicator {
-          .state-circle-hover(@big-dot-width, @big-dot-height, 0);
-          .state-circle-with-background-and-content-hover(@wz-color-default);
-        }
-
-        div {
-          min-height: @big-dot-height;
-        }
-      }
-
-      li.current .step-indicator {
-        .state-circle-with-background-and-content(@wz-color-current);
-      }
-      li.done .step-indicator {
-        .state-circle-with-background-and-content(@wz-color-done);
-      }
-      li.optional .step-indicator {
-        .state-circle-with-background-and-content(@wz-color-optional);
-      }
-      li.editing .step-indicator {
-        .state-circle-with-background-and-content(@wz-color-editing);
-      }
-
-      li.current a:hover .step-indicator {
-        .state-circle-with-background-and-content-hover(@wz-color-current);
-      }
-      li.done a:hover .step-indicator {
-        .state-circle-with-background-and-content-hover(@wz-color-done);
-      }
-      li.optional a:hover .step-indicator {
-        .state-circle-with-background-and-content-hover(@wz-color-optional);
-      }
-      li.editing a:hover .step-indicator {
-        .state-circle-with-background-and-content-hover(@wz-color-editing);
-      }
 
-      // default steps shouldn't change when hovered, because they aren't clickable
-      li.default a:hover .step-indicator {
-        .state-circle-with-background-and-content-nohover(@wz-color-current);
-      }
+    a:hover .step-indicator {
+      .state-circle-hover(@width, @height, @border);
+      .state-hover(@wz-color-default);
+    }
+    &.current a:hover .step-indicator {
+      .state-hover(@wz-color-current);
+    }
+    &.done a:hover .step-indicator {
+      .state-hover(@wz-color-done);
+    }
+    &.optional a:hover .step-indicator {
+      .state-hover(@wz-color-optional);
+    }
+    &.editing a:hover .step-indicator {
+      .state-hover(@wz-color-editing);
     }
   }
 
-  &.large-empty-symbols {
-    ul.steps-indicator {
-      padding: (@distance-between-steps / 2) (@distance-between-steps / 2) (@distance-between-steps / 2) (@distance-between-steps / 2 + @dot-baseline-distance + @big-dot-height);
-
-      li {
-        &:not(:last-child):before {
-          .line(@big-dot-width, @big-dot-height, @wz-color-default);
-        }
+  // default steps shouldn't change when hovered, because they aren't clickable
+  li.default a:hover .step-indicator {
```

Comment:
Sometimes we're only selecting a css class, e.g.  `.step-indicator`, while we're some other times selecting an element and a css class, e.g. `ul.steps-indicator`.
I think it would be best if we try to always use a combination of an element and a css class, because it makes it easier to identify on which element type a css class is applied.
What do you think?

Reply:
> I think it would be best if we try to always use a combination of an element and a css class, because it makes it easier to identify on which element type a css class is applied.

True, unless the element type is `div` or `span`, which carry little meaning. I tend to think of `div`'s and `span`'s as placeholders to which some meaning is assigned by way of CSS classes, so adding `div` or `span` to CSS selectors, in my opinion, has little sense. The code is good as it is :)